tests/nosetests/pyanaconda_tests/__init__.py

```diff
@@ -239,3 +239,27 @@ def patch_dbus_get_proxy(func):
     This is a shortcut to avoid creating of DBus proxies using DBus.
     """
     return patch('pyanaconda.dbus.DBus.get_proxy')(func)
+
+
+def set_up_dbus_container_mock(container_mock, paths_dict):
```

Comment:
I don't believe that this function is useful.  DBus containers can be used in unit tests decorated with `patch_dbus_publish_object`. They do all the mapping and generating of DBus paths, they just don't publish anything on DBus.

Reply:
Fixed by removing.